app/grandchallenge/components/templates/components/civ_set_form.html

```diff
@@ -69,8 +85,10 @@ <h3>
 {% block script %}
     {{ block.super }}
     {{ form_media }}
-    <script type="module" src="{% static 'components/js/create_extra_interfaces.mjs' %}" type="text/javascript"></script>
-    <script type="module" src="{% static 'components/js/autocomplete_htmx.mjs' %}" type="text/javascript"></script>
+    <script type="module" src="{% static 'components/js/create_extra_interfaces.mjs' %}"></script>
+    <script type="module" src="{% static 'components/js/autocomplete_htmx.mjs' %}"></script>
+    <script type="text/javascript" src="{% static "components/js/htmx/enable-me.js" %}"></script>
+    <script type="text/javascript" src="{% static "components/js/htmx/hide-me.js" %}"></script>

```

Comment:
New JS files should be modules (`type="module") and use ".mjs".

Reply:
Now done.